playbook/app/pb_kits/playbook/pb_selectable_card/selectable_card.rb

```diff
@@ -25,7 +25,7 @@ class SelectableCard < Playbook::KitBase
 
       def classname
         [
-          generate_classname_without_spacing("pb_selectable_card_kit", checked_class, enable_disabled_class),
+          generate_classname_without_spacing("pb_selectable_card_kit", checked_class, enable_disabled_class, display_input_class),
```

Comment:
As a rule of thumb, we don't change the default classname on a kit/variant... because it can affect custom css overrides in nitro-web, especially kits that are a part of forms. Example: https://github.com/search?q=repo%3Apowerhome%2Fnitro-web%20pb_selectable_card_kit_checked_enabled&type=code



Here's the way i "might" handle it (test and see, and look at other kit .rb files): 

```suggestion
          generate_classname_without_spacing("pb_selectable_card_kit", checked_class, enable_disabled_class) + display_input_class,
```

Reply:
I've changed it from:
`pb_selectable_card_kit_checked_enabled_display_input`
to
`pb_selectable_card_kit_checked_enabled display_input`